backend/inspirehep/records/api/literature.py

```diff
@@ -327,7 +327,6 @@ def add_file(
             )
         return {
             "key": new_key,
-            "original_url": original_url or url,
```

Comment:
in workflows will work, what about migration? Is ``original_url`` populated correctly? cc @michamos

Reply:
yeah, I had also checked the workflows, we populate the ```original_url``` there. Not sure about the migration though